.dockerignore

```
@@ -1,16 +1,33 @@
 Dockerfile
-Dockerfile-dev
-.circleci
-.vscode
+
+# repo files
+README.md
+CHANGELOG.md
+CONTRIBUTING.md
+schema1.svg
+.github
+
+/scripts
 /examples
-/test
-dev
-mesg-dev
-.git
+
+# test & debug files
+**/*_test.go
 **/*.test
 **/debug
 **/node_modules
-bin/cli
-docs/
-docker-images/
-.tendermint-validator/
\ No newline at end of file
+/docker-images
```

ignore bin folder to avoid sending to docker previously locally build bin
```suggestion
/bin
/docker-images
```